src/recorder.js

```diff
@@ -24,6 +24,7 @@ var Recorder = function( config ){
   this.config.encoderApplication = config.encoderApplication || 2049;
   this.config.encoderFrameSize = config.encoderFrameSize || 20;
   this.config.resampleQuality = config.resampleQuality || 3;
+  this.config.encoderComplexity = config.encoderComplexity || 9;
```

Comment:
Hmm. Some [doc](https://wiki.xiph.org/Opus_Recommended_Settings) states that the default complexity is maximum (10). I would prefer by default not to presume, and let the opus default value be selected. This is how we handle `bitRate` ATM.